Source/WebCore/animation/AnimationEffect.cpp

```diff
@@ -29,11 +29,13 @@
 #include "CSSAnimation.h"
 #include "CSSNumericFactory.h"
 #include "CSSNumericValue.h"
+#include "CSSParserContext.h"
```

Comment:
Not clear why this is relevant to this patch. If this addressing a unified build issue, perhaps it would be better to address this first as a standalone commit. I get the feeling this patch contains several of those. At the very least, these should be called out in the ChangeLog.

Reply:
Moved build fixes to https://github.com/WebKit/WebKit/pull/39640.